features/product/viewing_products/access_simple_product_show_page_from_products_index.feature

```diff
@@ -0,0 +1,90 @@
+@viewing_products
+Feature: Accessing the simple product show page from product index
+    In order to view detailed product information
+    As a Administrator
+    I want to be able to view product show page
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has "Over sized" and "Standard" shipping category
+        And the store has a tax category "No tax" with a code "nt"
+        And the store classifies its products as "Shield" and "Equipment"
+        And the store has a product "Iron shield" priced at "$20.00"
+        And the product "Iron shield" has orginal price "$25.00"
+        And there are 4 units of product "Iron shield" available in the inventory
+        And the "Iron shield" product has an image "mugs.jpg" with "main" type
+        And product "Iron shield" have the code equals "123456789"
+        And it belongs to "No tax" tax category
+        And product "Iron shield" has height "10.0", width "15.0", depth "20.0", weight "25.0"
+        And the product "Iron shield" has the slug "iron-shield"
+        And product "Iron shield" has a main taxon "Equipment"
+        And this product belongs to "Shield"
+        And this product belongs to "Over sized" shipping category
+        And the description of product "Iron shield" should be "Shield created by dwarf"
+        And the meta keywords of product "Iron shield" should be "shield"
+        And the short description of product "Iron shield" should be "good shield"
+        And this product has text attribute "serial number" with value "987654"
+        And the store has "Similar" and "dwarf equipment" product association types
+        And the store has a "Glass shield" product
+        And the product "Iron shield" has an association "Similar" with product "Glass shield"
+        And I am logged in as an administrator
+        And I browse products
+
+    @ui
+    Scenario: Viewing a simple product show page
+        When I access "Iron Shield" product page
+        Then I should see product show page without variants
+        And I should see product name "Iron shield"
+
+    @ui
+    Scenario: Viewing pricing block
+        When I access "Iron Shield" product page
+        Then I should see price "$20.00" for channel "United States"
+        And I should see original price "$25.00" for channel "United States"
+
+    @ui
+    Scenario: Viewing details block
+        When I access "Iron Shield" product page
+        Then I should see product's code "123456789"
+        And I should see product's channels "United States"
```

Comment:
I'd make s in `channels` optional here since the product is enabled for only one channel here. Also, this step sounds a bit unintuitive - `And I should see the product is enabled for channel "United States"` would be more natural.